temel_bilgiler/html_css/flexbox/flexbox-hizalama.md

```diff
@@ -0,0 +1,96 @@
+
+### Hizalama(Alignment)
+
+Bir örneğe bakalım.
+
+<p class="codepen" data-height="400" data-default-tab="html,result" data-slug-hash="MWoyBzR" data-editable="true" data-user="TheOdinProjectExamples" style="height: 400px; box-sizing: border-box; display: flex; align-items: center; justify-content: center; border: 2px solid; margin: 1em 0; padding: 1em;">
+  <span><a href="https://codepen.io">CodePen</a>'deki TheOdinProject (<a href="https://codepen.io/TheOdinProjectExamples">@TheOdinProjectExamples</a>) tarafından sağlanan <a href="https://codepen.io/TheOdinProjectExamples/pen/MWoyBzR"> 
+  flex-alignment örneğine</a> 
+  bakın.</span>
+</p>
+<script async src="https://cpwebassets.codepen.io/assets/embed/ei.js"></script>
+
+Şimdiye kadar öğrendiklerinizle, `.item` sınıfına `flex: 1` yazarsanız ne olacağını tahmin edebilmeniz gerekiyor. Konuya devam etmeden önce yapmaya çalışın!
+
+`.item` sınıfına `flex: 1` eklemek, öğelerin her birinin kullanılabilir alanı dolduracak şekilde büyümesini sağlar, ancak bunların aynı genişlikte kalmasını ama konteyner içinde kendilerini farklı şekilde dağıtmalarını istesek ne olur? Bunu yapabiliriz!
+
+`.item` sınıfından `flex: 1`'i kaldırın ve `.container`(.item sınıfının da içerisinde bulunduğu konteyner) sınıfına `justify-content: space-between`'i ekleyin. Bunu yapmak size şöyle bir şey vermelidir:
+
+![space between](https://cdn.statically.io/gh/TheOdinProject/curriculum/495704c6eb6bf33bc927534f231533a82b27b2ac/html_css/v2/foundations/flexbox/imgs/07.png)
+
+`justify-content` değeri, öğeleri **ana eksen** üzerinde hizalar. Burada kullanabileceğiniz birkaç değer var. Geri kalanını okuma ödevlerinde öğreneceksiniz, ancak şimdilik, kutuları ana eksen'de ortalayacak `center`(merkez) değerini kullanarak değiştirmeyi deneyin.
+
+**Çapraz eksen**'de öğelerin yerleşimini değiştirmek için `align-items` değerini kullanın. `.container`a `align-items: center` ekleyerek kutuları konteynerin merkezine getirmeyi deneyin. İstenen sonuç şöyle görünür:
+
+![centered](https://cdn.statically.io/gh/TheOdinProject/curriculum/495704c6eb6bf33bc927534f231533a82b27b2ac/html_css/v2/foundations/flexbox/imgs/08.png)
+
+`justify-content` ve `align-items` konteynerinizin ana ve çapraz eksenini temel aldığından, bir flex-container'ın flex-direction'nını değiştirdiğinizde davranışları değişir. Örneğin, `flex-direction`'ı `column` olarak değiştirdiğinizde, `justify-content` dikey olarak hizalar ve `align-items` yatay olarak hizalar. Bununla birlikte, en yaygın davranış varsayılandır, yani `justify-content` öğeleri yatay olarak hizalar (çünkü ana eksen varsayılan olarak yataydır) ve `align-items` öğeleri dikey olarak hizalar. Yeni başlayanların flexbox ile en çok karşılaştığı zorluklardan biri, bu davranış değişikliğidir.
+
+`justify-content` ve farklı özelliklerinin nasıl davrandığına dair etkileşimli bir demo için bu kısa video(Scrim)'ya göz atın:
```

Comment:
```suggestion
`justify-content` ve farklı özelliklerinin nasıl davrandığına dair etkileşimli bir demo için aşağıdaki Scrim'e göz atın:
```

Reply:
Burada, metindeki ilk `Scrim `kullanımı olduğu için ilk başta `kısa video` olarak çevirdim ve diğer `Scrim` kullanımlarında orjinal halini kullandım. Bu şekilde kullanılması daha doğru olmaz mı?

Reply:
Kesinlikle cok haklisin, dun scrimba'ya baktigimda video olmadigini dusunmustum. daha cok kodu deneyebilecegi bi editor ve kendi yapiyor her seyi sandim. Cok tesekkur ederim dikkatin icin 🤘🏻

Reply:
Ben Scrimba sayesinde bu yolculuğa başladım o yüzden biliyorum. Scrim aslında hem bir kısa video hem de kod editor yani ikisinin bir arada yapılabildiği bir araç, bu da Scrimba'ya özgü bir özellik zaten.

Reply:
Super, cok tesekkur ederim!